modules/csvtk/concat/main.nf

```diff
@@ -24,21 +24,22 @@ process CSVTK_CONCAT {
     val out_format
 
     output:
-    tuple val(meta), path("*.${out_format}"), emit: csv
-    path "versions.yml"                     , emit: versions
+    tuple val(meta), path("${prefix}.{tsv,csv}"), emit: csv
+    path "versions.yml"                         , emit: versions
```

Comment:
```suggestion
    tuple val(meta), path("*.${out_extension}"), emit: csv
    path "versions.yml"                       , emit: versions
```

Reply:
I wonder if prefeix should be included to prevent capturing other CSVs/TSVs

```
cat 1.csv 2.csv 3.csv > 4.csv
```

Reply:
Ah, ok. If more than one file has the same prefix then yes include it.